library/std/src/sys/configure_builtins.rs

```diff
@@ -0,0 +1,27 @@
+/// Hook into .init_array to enable LSE atomic operations at startup, if
+/// supported.
+#[cfg(all(
+    target_arch = "aarch64",
+    target_os = "linux",
+    any(target_env = "gnu", target_env = "musl"),
```

Comment:
You don't need to check target-env at all here. Any linux target should be fine.

Reply:
Ok, removed.